src/components/EventCard.js

```diff
@@ -13,6 +15,21 @@ const EventCard = ({ row }) => {
   const featureImage = row.values.feature_image;
   const eventImage = featureImage ? featureImage.url : '';
 
+  function checkAttendanceType(type) {
```

Comment:
If an event doesn't have a value for this column the app crashes. I think we should support missing values, have a default value, or make this column "optional" and just don't display the type

Reply:
Thanks for catching this, @anthmatic. I'll make the column "optional".

Reply:
That helped! Missing value still breaks everything, though, so I think we still need to check the event values.

Long term, I think it would make sense to come up with a pattern for this because I think this is an issue for most columns.

Reply:
Hi @anthmatic, could you clarify "Missing value still breaks everything"? I'm a little unclear.
In any case, I plan to work on this PR today and test out each column to see exactly which missing values break the app. 

I've also been working on the documentation for the app, and I do think that some columns in the HubDB table should be "required," eg. the event title, description, etc. Perhaps a solution for this would be to include warning messages if required columns are left "blank."

Reply:
Yeah, sorry... if an event is missing an attendance_type value, the whole app crashes.

Required fields does sound good, but HubDB doesn't support those, so your idea of having a warning message seems like the way to go. That way we can keep the app running and log an error to console or wherever makes sense.

Reply:
@anthmatic, I believe I've corrected for the app crashing if the attendance_type value is missing. I also deleted the attendance_type column entirely from the HubDB table to make sure: http://akemmerle-7428138.hs-sites.com/events

Reply:
Two strategies that I think could work are:

1. Default values for columns. For example, maybe type should default to `virtual`.
2. Filtering rows that are missing data and showing some sort of warning when the page is being edited (and perhaps previewed).

Reply:
@gcorne and I discussed this; I'll address this in a separate PR.